instructor/function_calls.py

```diff
@@ -119,9 +119,12 @@ def parse_anthropic_tools(
         validation_context: Optional[Dict[str, Any]] = None,
         strict: Optional[bool] = None,
     ) -> BaseModel:
-        tool_call = [c.input for c in completion.content if c.type == "tool_use"][0]
+        tool_calls = [c.input for c in completion.content if c.type == "tool_use"]
+        tool_call = tool_calls[0] if tool_calls else {}
```

Comment:
we should throw an error there right?

like assert len(tool_calls) == 1, "Please call exactly one tool"

Reply:
I left the error indication here:
https://github.com/jxnl/instructor/blob/230276e7067ec232448ed7394c95d1b697a87173/instructor/retry.py#L84
By passing an empty object, the language model will also understand what fields are missing

Reply:
But it may miss models whose fields all has a default value, let me change that